Sources/Rendering/OpenGL/VolumeMapper/index.js

```diff
@@ -568,6 +580,30 @@ function vtkOpenGLVolumeMapper(publicAPI, model) {
       // specify the planes in view coordinates
       program.setUniform3f(`vPlaneNormal${i}`, normal[0], normal[1], normal[2]);
       program.setUniformf(`vPlaneDistance${i}`, dist);
+
+      if (actor.getProperty().getUseLabelOutline()) {
+        const image = model.currentInput;
+        const worldToIndex = image.getWorldToIndex();
+
+        program.setUniformMatrix('vWCtoIDX', worldToIndex);
+
+        // Compute the view coordinate to world coordinate transformation matrix.
+        model.viewToWorld = mat4.create();
+        mat4.invert(model.viewToWorld, keyMats.wcvc);
+
+        const size = publicAPI.getRenderTargetSize();
+        const aspect = size[0] / size[1];
+        const viewToWorld = ren
+          .getActiveCamera()
+          .getCompositeProjectionMatrix(aspect, -1.0, 1.0);
+
```

Comment:
viewToWorld here I think is really world to device. Which is stored already in keyMats.wcdc

Reply:
The naming conventions throughout vtkjs were the biggest source of confusion for me. I completely agree with you, but was trying to stay consistent with other parts of the codebase, exactly the same code is named `viewToWorld` here:

https://github.com/Kitware/vtk-js/blob/a5b4567c8a5b8eec4546eacf9c20045caeab2022/Sources/Rendering/Core/Renderer/index.js#L241

I've checked, and inverting keyMats.wcdc gives me the same result, thanks!